recipes-mender/mender/mender_0.1.bb

```diff
@@ -37,7 +37,7 @@ do_compile() {
   export PATH
 
   cd ${S}
-  godep go build -o ${B}/mender
+  oe_runmake GO='godep go'
```

Comment:
We might need to use pure go here after https://github.com/mendersoftware/mender/pull/62, but I'm not sure yet, so we can leave it for now. Otherwise a good change.